### DISCUSSION TITLE
Help with extending the project

### DISCUSSION THREAD

**Discussion by loaishawar** [TheAlphamerc/flutter_twitter_clone#197] [Q&A]
Hi i am very interested in your project twitter clone and i need your help to extend it to work as a following website stocktwits.com  and i am just wondering what would be the best way to redesigning the firebase and make the stocks as a primary , can you please help me to achieve that

**Comment by TheAlphamerc** [comment]
Hi @loaishawar, Sorry to say I can't help you with this as  I didn't know about stocktwits.com and how its work.
